powersoftwo.c

```diff
@@ -0,0 +1,20 @@
+/* Christopher Liu - I realized this could be done with a for loop instead. I'll include it too */
+#include <stdio.h>
+
+int main()
```

Comment:
Perfect job! Either one of these implementations would have been fine - great job thinking of two different ways to go about completing this assignment! 💯